product_docs/docs/biganimal/release/overview/updates.mdx

```diff
@@ -2,14 +2,25 @@
 title: Periodic maintenance
 ---
 
-EDB performs periodic maintenance to ensure stability and security of your clusters. We perform minor version upgrades and patch updates as part of this periodic maintenance. You're notified in the BigAnimal portal before maintenance occurs. Details are available on the [BigAnimal status page](https://status.biganimal.com/). You can subscribe to get these updates in a feed by selecting **Subscribe to Updates** on the status page. 
+EDB performs periodic maintenance to ensure stability and security of your clusters. We perform minor version upgrades and patch updates as part of this periodic maintenance. 
+
+## Notification of upcoming maintenance
+
+You're notified in the BigAnimal portal before maintenance occurs. Details are available on the [BigAnimal status page](https://status.biganimal.com/). You can subscribe to get these updates in a feed by selecting **Subscribe to Updates** on the status page. 
 
 EDB reserves the right to upgrade customers to the latest minor version without prior notice in an extraordinary circumstance. You can't configure minor versions. 
 
 In some cases, these updates might terminate existing network connections to your clusters. If that happens, the outage is typically less than 30 seconds. Be sure your applications are configured to automatically reconnect when connections are interrupted. Most modern database libraries do this by default. 
 
+## Specifying maintenance windows
+
 If you want to control when the updates are pushed, you can specify a weekly maintenance window for each cluster or each data group in the case of a distributed high-availability cluster. BigAnimal displays a *scheduled maintenance* message on your cluster list four hours prior to the scheduled maintenance time to remind you of the upcoming maintenance window. This reminder allows you to make any necessary preparations, such as saving your work and closing any open connections. For more information on specifying maintenance windows, see [Maintenance](/biganimal/latest/getting_started/creating_a_cluster/#maintenance). 
 
+## Maintenance for high-availability clusters
+
+For high-availability clusters, periodic maintenance is performed first on the standby replicas and then on the primary.
```

Comment:
@amrita-suresh this applies just to primary/standby high-availability clusters, correct? 
```suggestion
For primary/standby high-availability clusters, periodic maintenance is performed first on the standby replicas and then on the primary.
```